projects/common/src/lib/models/data-grid-pagination.model.ts

```diff
@@ -1,12 +1,18 @@
 
 export class DataGridPaginationModel {
 
+    public Length?: number;
```

Comment:
If you run across properties or functions without comments, go ahead and add them. Even if items are easy to understand, I think even small comments help with better understanding. I've been trying to catch up on missing comments as well.